src/Tizen.NUI.Components/Controls/Navigation/Navigator.cs

```diff
@@ -858,5 +861,33 @@ private void InitializeAnimation()
                 newAnimation = null;
             }
         }
+
+        // Register dialog(popup) to Accessibility bridge
+        internal void AddPageToList(Page page)
```

Comment:
I think we could do this for all content page,
and may be internal method is not necessary that only call it one place.
if it necessary,
function name would be better to use show/hide term that other developer can guess what this function actaully does.

thank you for update!